src/components/login/login.component.js

```diff
@@ -1,43 +1,48 @@
 import React from 'react';
-import { Formik, Field, Form, ErrorMessage } from 'formik';
 import * as Yup from 'yup';
+import { Formik, Field, Form, ErrorMessage, withFormik } from 'formik';
+import { bindActionCreators } from 'redux';
+import loginUserAction from './login.actions'
+import { connect } from 'react-redux';
+
 
 const initialValues = {
     email: '',
-    password: '' 
+    password: ''
 }
 
 const LoginSchema = Yup.object().shape({
     email: Yup.string()
-    .email("Invalid email"),
+        .email("Invalid email"),
     password: Yup.string()
-    .min(5, "Too Short!")
-    .max(50, "Too Long!")
+        .min(5, "Too Short!")
+        .max(50, "Too Long!")
 });
 
-const Login = () => (
+const LoginPage = (props) => (
```

Comment:
can you use object destructuring here for `prop`? Also, we will need to use `PropTypes` for every component that has props